config/routes.rb

```diff
@@ -96,8 +96,8 @@
     resources :tags, except: [:show]
 
     namespace :qa do
-      resources :issues, only: [:edit, :index, :show] do
-        collection { put :update }
+      resources :issues, only: [:edit, :index, :show, :update] do
+        collection { put :bulk_update }
```

Comment:
I think it's time to bring Pro's :multiple_update  routing concern to CE. It'd also require a route/action rename from bulk_ to multiple_